web/app/plugins/cbf-academy/assets/js/admin/cbf-academy.js

```diff
@@ -0,0 +1,36 @@
+/******/ (function() { // webpackBootstrap
+var __webpack_exports__ = {};
+/* eslint-disable */ // Remove this line if customising this file
+(function ($) {
+  "use strict";
+
+  /**
+   * All of the code for your admin-facing JavaScript source
+   * should reside in this file.
+   *
+   * Note: It has been assumed you will write jQuery code here, so the
+   * $ function reference has been prepared for usage within the scope
+   * of this function.
+   *
+   * This enables you to define handlers, for when the DOM is ready:
+   *
+   * $(function() {
+   *
+   * });
+   *
+   * When the window is loaded:
+   *
+   * $( window ).load(function() {
+   *
+   * });
+   *
+   * ...and/or other possibilities.
+   *
+   * Ideally, it is not considered best practise to attach more than a
+   * single DOM-ready or window-load handler for a particular page.
+   * Although scripts in the WordPress core, Plugins and Themes may be
+   * practising this, we should strive to set a better example in our own work.
+   */
+})(jQuery);
+/******/ })()
+;
```

Comment:
Missing empty line at end of this file

Reply:
It’s a generated file. Will look at adding a lint step after building, but don’t want to hold up this PR for that.